AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -92,6 +92,9 @@ techadvisor.co.uk,forum.notebookreview.com,computerworld.com,skysports.com#$#bod
 !
 !
 lazimbana.com##.lazim-cookie-alert
+kontrast.dk###modal-background
+kontrast.dk,sued.aldi-liefert.de,enirocentrum.pl,felix1.de,goldenline.pl,panoramafirm.pl,hofer.at,dizioneri.com,m.n11.com,wuerth.de,greenweez.com,deutsches-ausschreibungsblatt.de,sued.aldi-liefert.de,capveriant.com,calkalasutasayrani.com,xn--giant-dsseldorf-5vb.de##.modal-backdrop
+kontrast.dk#$#body { overflow: auto !important; }
```

Comment:
For `kontrast.dk`.

```
kontrast.dk#$##modal-background { display: none !important; }
kontrast.dk#$#.modal-backdrop { display: none !important; }
kontrast.dk#$#body { overflow: visible !important; }
```

Reply:
Sorry. I was gone for a while. Btw what's the difference? My rules work too.

Reply:
CB platforms don't have support `#$#` rules, so you can block this, but the page scrolling will be broken.
That's why we don't use `##` rules in such cases.

Reply:
Okay. Why don't the Content Blockers have JS, Sscriptlets and ext css support?